Add Anime topic (#3666)
github/explore: topics/anime/index.md

---
aliases: animes
display_name: Anime
short_description: Anime is hand-drawen & computer-generated animation produced in Japan.
topic: anime
wikipedia_url: https://en.wikipedia.org/wiki/Anime
---

> Anime (Japanese: アニメ, IPA: [aɲime] (listen)) is hand-drawn and computer-generated animation originating from Japan. Outside of Japan and in English, anime refers specifically to animation produced in Japan.

Reviewer: >anime refers specifically to animation produced in Japan.

I'm not sure this is true. From the existing topic page alone, there are tools that help you produce anime, which can be made from anywhere. Thoughts on removing this line?
```suggestion
Anime (Japanese: アニメ, IPA: [aɲime] (listen)) is hand-drawn and computer-generated animation often originating from Japan.
```
